styles.css

```diff
@@ -0,0 +1,427 @@
+body {
+    margin: 0;
+    padding: 0;
+    font-family: "Kaushan Script", cursive;
+}
+
+:root {
+    --pink: #ef7998;
+    --mainYellow: #f9e494;
+    --mainWhite: #fff;
+    --mainBlack: #000;
+    --yellowTrans: #f9e49480;
+    --mainGrey: #eeeeee;
+}
+
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 1rem 2rem;
+}
+
+.navbar {
+    position: absolute;
```

Comment:
Not sure why we need this for the navbar. it can be centered in it's div Container